release/models/gnsi/openconfig-gnsi-pathz.yang

```diff
@@ -109,62 +115,44 @@ module openconfig-gnsi-pathz {
     description
       "A collection of counters collected by the gNSI.pathz module.";
 
-    container gnmi-pathz-policy-counters {
+    container policies {
       config false;
       description
-        "A collection of per-OpenConfig path counters.";
-
-      uses gnmi-pathz-policy-xpath-success-failure-counters;
-    }
-  }
-
-  grouping gnmi-pathz-policy-xpath-success-failure-counters {
-    description
-      "A collection of per-OpenConfig path counters.";
-
-    container paths {
-      description
-        "Container for a collection of per-OpenConfig path counters.";
+        "Container for a collection of per-Pathz policy counters.";
 
-      list path {
```

Comment:
If it's implemented, rather than replace the paths container entirely, you can mark it as `deprecated`.  If not, then this LGTM.